src/Codeception/SuiteManager.php

```diff
@@ -60,14 +60,15 @@ class SuiteManager
      * @var string
      */
     protected $path = '';
+
     protected $printer = null;
 
     protected $env = null;
```

Comment:
This class had suspiciously many untyped fields, so I checked them and it appeared that most of them are actually unused and can be removed.
Unused fields:
```
    public static $environment;
    public static $name;
    protected $testLoader;
    protected $tests = [];
    protected $debug = false;
    protected $printer = null;
```

`protected $path = '';` is assigned a value in constructor, but it is never used anywhere.

Reply:
I couldn't find `protected $testLoader;` in this class.
I also typed `protected $env` as a string